env/local.env

```diff
@@ -4,6 +4,13 @@ ASSET_INVENTORY_API_URL=http://example.com/asset-inventory/v1
 # TODO: Add ACCOUNTS env var. If present, it should be used instead of the
 # ASSET_INVENTORY_API_URL.
 
+# Defines a comma separated list of accounts to scan, when defined the Asset
+# Inventory is not queried.
+ACCOUNTS_TO_SCAN = 123456789012,123456789013
```

Comment:
What about renaming the config parameter to `ACCOUNTS`? It is simpler and descriptive. If you accept this suggestion, please update also the PR description.